docs/2-tutorials/2-develop/9-developing-with-wikitude.html.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Wikitude and Augmented Reality
+url: tutorials/develop/wikitude
+github_url: https://github.com/phonegap/phonegap-docs/blob/wikitude/docs/2-tutorials/2-develop/9-developing-with-wikitude.html.md
+layout: subpage
+---
+
+## Overview
+
+Wikitude's augmented reality Cordova plugin has now come to the PhoneGap Developer App. Users of the PhoneGap Developer App will now be able to detect images or locations and project 2D images or 3D models.
```

Comment:
Wikitude's augmented reality Cordova plugin has recently become available in the PhoneGap Developer App, allowing developers to build apps that can detect images or locations and project 2D images or 3D models.